index.d.ts

```diff
@@ -1061,6 +1080,13 @@ declare module '@xmldom/xmldom' {
 
 	interface ProcessingInstruction extends Node {
 		nodeType: typeof Node.PROCESSING_INSTRUCTION_NODE;
+		/**
+		 * Everything that goes after the target, excluding `?>`.
+		 * [MDN Reference](https://developer.mozilla.org/docs/Web/API/CharacterData/data)
+		 */
+		data: string;
+		/** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ProcessingInstruction/target) */
```

Comment:
Use definition provided by MDN here too: A name identifying the application to which the instruction is targeted.